castai/data_source_gke.go

```diff
@@ -8,13 +8,37 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+const (
+	// GKEPoliciesResourceName is the name of the resource
+	GKEPoliciesResourceName = "policy"
+	// GKEFeaturesResourceName is the name of the policies per feature
+	GKEFeaturesResourceName                        = "features"
+	GKELoadBalancersNetworkEndpointGroupFeature    = "load_balancers_network_endpoint_group"
+	GKELoadBalancersTargetBackendPoolsFeature      = "load_balancers_target_backend_pools"
+	GKELoadBalancersUnmanagedInstanceGroupsFeature = "load_balancers_unmanaged_instance_groups"
 )
 
 func dataSourceGKEPolicies() *schema.Resource {
 	return &schema.Resource{
 		ReadContext: dataSourceGKEPoliciesRead,
```

Comment:
Let's add description? Right now it has none. It would be nice and it would be updated in generated documentation as well.
Something like this:
```suggestion
		ReadContext: dataSourceGKEPoliciesRead,
		Description: "Get list of required GCP policies for GKE cluster onboarding", 
```